databricks_cli/accounts/api.py

```diff
@@ -0,0 +1,130 @@
+"""Implement Databricks Accounts API, interfacing with the AccountsService."""
+# Databricks CLI
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"), except
+# that the use of services to which certain application programming
+# interfaces (each, an "API") connect requires that the user first obtain
+# a license for the use of the APIs from Databricks, Inc. ("Databricks"),
+# by creating an account at www.databricks.com and agreeing to either (a)
+# the Community Edition Terms of Service, (b) the Databricks Terms of
+# Service, or (c) another written agreement between Licensee and Databricks
+# for the use of the APIs.
+#
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from databricks_cli.sdk import AccountsService
+
+
+class AccountsApi(object):
+    """Implement the databricks '2.0/accounts' API Interface."""
+
+    def __init__(self, api_client):
+        self.client = AccountsService(api_client)
+
+    def create_credentials(self, account_id, json):
+        """Create a new credentials object for the AWS IAM role reference."""
+        if account_id is None:
+            raise TypeError('Expected account_id')
+        endpoint = "/accounts/%s/credentials" % (account_id)
+        return self.client.client.perform_query('POST', endpoint, data=json)
```

Comment:
Why not write these into the client SDK as well?

Reply:
I did not see any other POST perform_query methods accepting a JSON for other objects in the client SDK, so I skipped these too.